README.md

```diff
@@ -69,6 +69,14 @@
 
 ## Calendar
 
+- [Class Schedule Viewer](https://github.com/rushhiii/Scriptable-IOSWidgets/tree/main/Class%20Schedule%20Viewer) - Displays your daily or weekly class schedule from a Google Sheet.
```

Comment:
You have submitted a lot of widgets. I like to keep a fair spread of developers. In my opinion this one can be removed, since very similar functionality can be archived with iOS' calender widget. Do you mind?